db/migrate/20160831015641_remove_has_muted_from_users.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveHasMutedFromUsers < ActiveRecord::Migration
+  def change
+    User.where(has_muted: true).each do |user|
+      user.experienced!("threadMuted")
```

Comment:
Happy to change, was just thinking about how you would say it aloud, e.g. _user experienced mutingThread_